connector_helpscout/models/helpscout_conversation/importer.py

```diff
@@ -70,18 +62,18 @@ def stage_id(self, record):
 
     @mapping
     def tag_ids(self, record):
-        tags = self.env['project.tags'].search([
-            ('name', 'in', record.tags)
-        ])
+        tags = self.env['project.tags'].search([('name', 'in', record.tags)])
```

Comment:
You should probably search on `helpscout.project.tags`, and add `backend_id` to the domain. Otherwise all tags are included, which will result in an infinite recursion if a tag is added from outside of this connector.